app/views/layouts/_lte_sidebar.html.erb

```diff
@@ -96,7 +96,7 @@
       </li>
       <li class="nav-item <%= 'active' if current_page?(broadcast_announcements_path) %>">
         <%= link_to(broadcast_announcements_path, class: "nav-link #{active_class(['broadcast_announcements'])}") do %>
-          <i class="nav-icon fa fa-circle-o"></i>Partner Announcement 
+          <i class="nav-icon fa fa-circle-o"></i>Partner Annoucement
```

Comment:
Introduced a typo?

Reply:
Fixed it in [b73877c](https://github.com/rubyforgood/human-essentials/pull/3598/commits/b73877ca46991edb83b8cc18e9926277269c34b4)